packages/ERTP/src/amountMath.js

```diff
@@ -143,11 +144,11 @@ const optionalBrandCheck = (allegedBrand, brand) => {
 };
 
 /**
- * @template {AssetKind} [K=AssetKind]
+ * @template {AssetKind} [K=AssetKind] Default is `AssetKind`
```

Comment:
this is the most surprising change. We can disable it with `jsdocAddDefaultToDescription` (Added in https://github.com/hosseinmd/prettier-plugin-jsdoc/commit/c3fb1680d9fd20678d4fb4069395e14fc0440e19#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R124 but not documented in https://github.com/hosseinmd/prettier-plugin-jsdoc#options)

Reply:
Does it add any value? If not, yes, please do disable it.